src/style.css

```diff
@@ -1,3 +1,126 @@
-body {
+:root {
   margin: 0;
+  padding: 0;
+
+  font-family: Roboto, Helvetica, sans-serif;
+  --secondary-color: #616070;
+  --blue-accent: #00acdc;
+  color: #060b35;
+}
+
+p {
+  margin: 0;
+}
+
+.card {
```

Comment:
Don't add external styles (positioning or margins) to BEM-blocks. Use mix where necessary and move all external styles under element selector.